src/modules/addEvent.js

```diff
@@ -0,0 +1,75 @@
+const input = document.querySelector('.input-event');
+const list = document.querySelector('.list');
+const ellipse = document.querySelectorAll('.fa-ellipsis-vertical');
+
+const data = JSON.parse(localStorage.getItem('data')) || [];
+
+// ToDoList object class
+class EventObj {
+  constructor(task, done, index) {
+    this.task = task;
+    this.done = done;
+    this.index = index;
+  }
+
+  // Create event list
+   static createEvent = () => {
+     const showEvent = data.map((e, i) => `<li><input type="checkbox" class="tick"><input type="text" class="list-input" id="edit" value="${e.task}" >
+    <i class="fa-solid fa-ellipsis-vertical"></i><i id="${i}" class="fa-solid fa-trash-can delete"></i></li>`).join('');
+     list.innerHTML = showEvent;
+
+     const del = document.querySelectorAll('.delete');
+     del.forEach((e, i) => {
+       e.addEventListener('click', () => {
+         EventObj.delEvent(i);
+       });
+     });
+     const tick = document.querySelectorAll('.tick');
+     // Edit added event on UI
+     const listData = document.querySelectorAll('.list-input');
+     listData.forEach((e, i) => {
+       e.addEventListener('change', () => {
+         data[i].task = e.value;
+         localStorage.setItem('data', JSON.stringify(data));
+       });
+     });
+
+     for (let i = 0; i < ellipse.length; i += 1) {
+       ellipse[i].addEventListener('click', () => {
+         del[i].style.display = 'inline';
+         ellipse[i].style.display = 'none';
+       });
+     }
```

Comment:
I would suggest you use array method like 'forEach' to loop through the ellipse instead of traditional for loop

Reply:
@Fayob Thanks 👍